packages/desktop-client/src/components/modals/ScheduledTransactionMenuModal.tsx

```diff
@@ -33,24 +33,25 @@ export function ScheduledTransactionMenuModal({
     borderTop: `1px solid ${theme.pillBorder}`,
   };
   const scheduleId = transactionId?.split('/')?.[1];
-  const scheduleData = useSchedules({
-    transform: useCallback(
-      (q: Query) => q.filter({ id: scheduleId }),
+  const { isLoading: isSchedulesLoading, schedules } = useSchedules({
+    query: useMemo(
+      () => q('schedules').filter({ id: scheduleId }).select('*'),
```

Comment:
nit: it's nice to have hooks at the top level so we don't accidentally make them conditional later

Reply:
Good idea. Updated :)